cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIPostgreSQLSink.java

```diff
@@ -254,7 +254,9 @@ public void configure(Context context) {
     @Override
     public void start() {
         try {
-            createPersistenceBackend(postgresqlHost, postgresqlPort, postgresqlUsername, postgresqlPassword, maxPoolSize, postgresqlDatabase, postgresqlOptions);
+            if (buildDBName(null) != null) {
```

Comment:
I would expect a else case with a warn log about dbname is null, make sense?

Reply:
I agree that there must be some mention that the sql backend will not be generated on startup as usuall, but I wasn't sure of how lo log it. Please checkout the text I ussed.

[Fixed in 84e286e](https://github.com/telefonicaid/fiware-cygnus/pull/1876/commits/84e286ec4d742ba729bf510e7711ecc58e8b2f24)